src/Symfony/Bundle/FrameworkBundle/Resources/config/notifier_transports.php

```diff
@@ -279,13 +279,17 @@
         ->set('notifier.transport_factory.simple-textin', Bridge\SimpleTextin\SimpleTextinTransportFactory::class)
             ->parent('notifier.transport_factory.abstract')
             ->tag('texter.transport_factory')
-    
+
         ->set('notifier.transport_factory.click-send', Bridge\ClickSend\ClickSendTransportFactory::class)
             ->parent('notifier.transport_factory.abstract')
             ->tag('texter.transport_factory')
-    
+
         ->set('notifier.transport_factory.smsmode', Bridge\Smsmode\SmsmodeTransportFactory::class)
             ->parent('notifier.transport_factory.abstract')
             ->tag('texter.transport_factory')
+
+        ->set('notifier.transport_factory.novu', Bridge\Novu\NovuTransportFactory::class)
```

Comment:
please move it up, to stay in alpha order, thanks

Reply:
@OskarStark  Where? there is basicly no alphabet order :sweat_smile: